### PR TITLE
Adding superblockify recipy

### PR DESCRIPTION
**Pull request by cbueth**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

Waiting since a week for a review, replaces #26047 #25984 due to renaming the branch to be merged.

### CODE REVIEW

**Comment by cbueth** [comment]
@conda-forge/help-python ready to be merged!

**Comment by cbueth** [comment]
Waiting to be merged! Earlier PR has been waiting for a week, with successful checks without being merged, while other similar PRs have been merged constantly. New PR only due to renaming the feature branch.
Is there something else missing to be reviewed? Please let us know. If nothing to comment, please merge.
@conda-forge/staged-recipes

**Comment by cbueth** [comment]
@xhochy has someone been assigned to this PR yet or can we get an ETA?

**Comment by xhochy** [comment]
This is a volunteer run effort and as you aren't paying anyone, there are no ETAs. Please be respectful.

**Comment by cbueth** [comment]
Thank you for the voluntary work. I will respect this.

File: recipes/superblockify/meta.yaml
Comment (BastianZim) [inline recipes/superblockify/meta.yaml:2]: This will have to be published as a prerelease software: https://conda-forge.org/docs/maintainer/knowledge_base/#pre-release-builds

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/superblockify/meta.yaml
Comment (cbueth) [reply recipes/superblockify/meta.yaml]: Thank you for highlighting this. I added the RC channel in the `conda_build_config.yaml`, see the last commit.

**Pull request merged**
